src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyNotModifiedResponse.java

```diff
@@ -153,6 +164,16 @@ private void onAbort(AbortFW abort)
                                abort.trace());
     }
 
+    private void onReset(ResetFW reset)
+    {
+        //NOOP
```

Comment:
Shouldn't we clean up `onReset` ?

Reply:
I believe there is nothing to clean up. Because once we send begin frame we are purging request slot.